src/main/java/org/jabref/gui/desktop/os/Linux.java

```diff
@@ -30,7 +30,11 @@ public void openFile(String filePath, String fileType) throws IOException {
             viewer = "xdg-open";
         }
         String[] cmdArray = { viewer, filePath };
-        Runtime.getRuntime().exec(cmdArray);
+        Process p;
+        p = Runtime.getRuntime().exec(cmdArray);
+        BufferedReader in = new BufferedReader(new InputStreamReader(p.getErrorStream()));
+        String line;
+        line = in.readLine();
```

Comment:
As @calixtus said, it would be nice if you could add a logging statement here:
Add `private static final Logger LOGGER = LoggerFactory.getLogger(Linux.class);` at the top of the class and then here `LOGGER.debug("Received output: " + line)`.

Reply:
OK. I have done the modifications. I had to add also the lines 
`import org.slf4j.Logger;
import org.slf4j.LoggerFactory;`
But I am not sure how I shall proceed now? How do I modify the fix I sent? A new pull request? 
Thank your for your help

Reply:
You just commit the changes on your branch and then push them to your fork.
the PR will be updated automatically

Reply:
OK. That is done. Thank you for your help!